lib/migration.js

```diff
@@ -99,15 +99,36 @@ function mixinMigration(PostgreSQL) {
       }
     };
 
-    async.series([
-      function(cb) {
-        applyPending([self.getAddModifyColumns, self.getDropColumns], cb);
-      },
-      function(cb) {
-        self.addIndexes(model, actualIndexes, cb);
-      },
-    ], function(err, result) {
-      cb(err, result[0]);
+    self.discoverForeignKeys(self.table(model), {}, function(err, actualFks) {
+      if (err) {
+        console.log('Failed to discover "' + self.table(model) +
+        '" foreign keys', err);
+      }
```

Comment:
Should we callback with `err` and return here?

Reply:
yes, I thought it might be aggressive to fail for an error looking up FKs, but I will cb with the error